conf/config.json

```diff
@@ -6,6 +6,19 @@
     "kafka": {
         "hosts": "127.0.0.1:9092"
     },
+    "s3": {
```

Comment:
I believe we still need a `transport` key. Not sure if you want to set that globally, too

Reply:
Yes, it's part of the config, but the default is "http" so it should be fine. You can access the transport key from the config object, it will be always populated by the joi parser even if not set in config.json (with the default value in such case).

Reply:
And this config.json is not used for deployments with Federation, since we generate a config from a template (see PR https://github.com/scality/Federation/pull/1315)